.github/workflows/api_doc_build.yml

```diff
@@ -0,0 +1,42 @@
+name: API docs build
+
+on:
+  push:
+    - branches:
+        - 'bagatur/api_docs_build_gha'
```

Comment:
this might create an infinite loop with the action at the end

Reply:
its commiting to api_docs_build, this is api_docs_build_gha